internal/config/config.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/influxdata/telegraf/plugins/inputs"
 	"github.com/influxdata/telegraf/plugins/outputs"
 
-	"github.com/naoina/toml"
+	"github.com/influxdata/config"
 	"github.com/naoina/toml/ast"
```

Comment:
still using naoina/toml here, is there not an equivalent for the ast tables in influxdata/config yet?